api/src/server.rs

```diff
@@ -356,6 +367,8 @@ pub struct Server<C, I, M> {
     insert_task: Arc<InsertTask>,
     marker: PhantomData<C>,
     authentication: bool,
+
+    pipeline: SessionContext,
```

Comment:
is it worth making this an option and just returning an error/bad request saying it's disabled if the context isn't set because the feature is off?